### DISCUSSION TITLE
Wasted hours realizing I cant use 1.5 loras. Why? What other UI can I use

### DISCUSSION THREAD

**Discussion by samboa86** [lllyasviel/Fooocus#1356] [General]
I (like most here) are new to this, tried using some Loras from civit.ai but turns out I cannot as they are 1.5. Didnt realize this after testing creating for hours. Cant even use a 1.5 base model, etc.
What other UI should I use so I can use the vast majority of civit files?

**Comment by lllyasviel** [comment]
Hi you can use any 1.5 loras in Fooocus.

Alternative UI is https://github.com/AUTOMATIC1111/stable-diffusion-webui

Besides, comments from blank github account may be removed to follow anti-spamming policy.

**Reply by samboa86** [reply]
how do I use 1.5 loras in Fooocus? I am trying with no success, doesnt do anything.
Currently I am getting a "Waiting for task to start ..." and cmd line not moving.

**Reply by lllyasviel** [reply]
see also https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md if you run into any problems
